vendor/modules.txt

```diff
@@ -320,6 +320,7 @@ k8s.io/apimachinery/pkg/runtime
 k8s.io/apimachinery/pkg/api/errors
 k8s.io/apimachinery/pkg/apis/meta/v1
 k8s.io/apimachinery/pkg/util/intstr
+k8s.io/apimachinery/pkg/types
```

Comment:
Is it ok to format this file? https://github.com/k8snetworkplumbingwg/kubemacpool/pull/113/files#r411325352

Reply:
it is auto generated after I performed go mod tidy (I updated after eliminating the leder election)